tests/shell/pipeline/commands/internal.rs

```diff
@@ -88,30 +74,24 @@ fn for_loop() {
 
 #[test]
 fn subexpression_handles_dot() {
-    Playground::setup("subexpression_handles_dot", |dirs, sandbox| {
-        sandbox.with_files(vec![FileWithContentToBeTrimmed(
-            "nu_times.csv",
-            "
-                name,rusty_luck,origin
-                Jason,1,Canada
-                JT,1,New Zealand
-                Andrés,1,Ecuador
-                AndKitKatz,1,Estados Unidos
-            ",
-        )]);
-
-        let actual = nu!(
-        cwd: dirs.test(), pipeline(
+    let sample = r#"
+                [[name, rusty_luck, origin];
+                 [Jason, 1, Canada],
+                 [JT, 1, "New Zealand"],
+                 [Andrés, 1, Ecuador],
+                 [AndKitKatz, 1, "Estados Unidos"]]
+            "#;
+
+    let actual = nu!(pipeline(&format!(
         "
-            echo (open nu_times.csv)
```

Comment:
based on the title this sounds like it should check for `(open nu_times.csv)` being handled correctly.

Else the whole construct with `echo (open ...)` doesn't make much sense.

Maybe the rest of the test can be simplified to make it a bit less confusing but else we should preserve that based on whatever forced us to add it in the first place.